src/index.js

```diff
@@ -0,0 +1,25 @@
+import getWeather from './getData';
+import { getCityForm, changeButton } from './documentBottom';
+import newCity from './newCity';
+import tempSymbol from './tempSymbol';
+
+function init() {
+  getWeather('Juiz de Fora');
+
+  const tmpSymbol = tempSymbol();
+
+  document.getElementById('weather-bottom').appendChild(getCityForm());
+  const button = document.createElement('div');
+  button.setAttribute('id', 'changeTempButton');
+  button.innerHTML = changeButton(tmpSymbol);
+  document.getElementById('weather-bottom').appendChild(button);
+  document.getElementById('submitCity').addEventListener('click', newCity);
+  document.getElementById('changeTempButton').addEventListener('click', () => {
+    tmpSymbol.setTmpSmbl();
+    const city = document.getElementById('cityName').innerHTML.split(',')[0];
+    getWeather(city, tmpSymbol.getTmpSmbl());
+    document.getElementById('changeTempButton').innerHTML = changeButton(tmpSymbol);
+  });
+}
```

Comment:
Here, `changeTempButton` is a `div` instead to be **`button`**. Kindly change change that.
```html
<div id="changeTempButton">
    <button id="changeTempSymbol" class="btn-black">
      Change to Celsius
    </button>
  </div>
```
According to your code HTML up there, the event listener `click` must the set up on **`#changeTempSymbol`**